R/validate-pds-trips.R

```diff
@@ -21,12 +21,9 @@ validate_pds_trips <- function(log_threshold = logger::DEBUG){
   pds_trips <- get_preprocessed_trips(pars)
 
   # remove duplicated trips
-  pds_trips <- dplyr::distinct(pds_trips, .data$Started, .data$Ended,.data$Boat,
-                               .data$`Boat Name`,.data$`Boat Gear`,.data$Community,
-                               .data$`Duration (Seconds)`,.data$`Range (Meters)`,
-                               .data$`Distance (Meters)`,.data$IMEI,
-                               .data$`Device Id`,.data$`Last Seen`,
-                               .keep_all = TRUE)
+  pds_trips <- pds_trips %>%
+    dplyr::arrange(dplyr::desc(Trip)) %>%
```

Comment:
Why need to arrange by Trip?

Reply:
Because `distinct()` keeps only the first row and we might want to keep the largest trip id (it is a latest version of the trip)